global.yaml

```diff
@@ -251,6 +251,12 @@ global:
   # the desired values.
   defaultNodeSelector: {}
 
+  # Default node tolerations to be applied to all deployments so that all pods can be 
```

Comment:
I don't think this makes any sense with the new installer - each component is installed independently, and it would have different settings.  We also want to move this kind of 'global override' to kustomize and reduce API surface in values.yaml

Reply:
remove global default value.